src/citrine/resources/gemtables.py

```diff
@@ -244,7 +246,12 @@ def get_by_build_job(self, job: Union[JobSubmissionResponse, UUID], *,
             The table built by the specified job.
 
         """
-        status = _poll_for_job_completion(self.session, self.project_id, job, timeout=timeout)
+        # TODO: Should this use the project or team version?
```

Comment:
Calling this out here in case I forget to ask Pablo before the PR.

Reply:
Not strictly required, but if it happens that you have the teamId it would be nice, I'm trying to move the jobs API to the team level.